scripts/scripts.js

```diff
@@ -111,6 +111,28 @@ export function addFavIcon(href) {
   }
 }
 
+function addBackToTop(main) {
+  const backToTop = document.createElement('div');
+  backToTop.setAttribute('id', 'back-to-top');
+
+  const link = document.createElement('a');
+  link.setAttribute('href', '#');
+  link.innerHTML = 'Go to Top';
+
+  backToTop.append(link);
+  main.append(backToTop);
+
+  document.addEventListener('scroll', (e) => {
+    // back to top only appears after a bit of scrolling
+    if (window.scrollY <= 200) {
```

Comment:
might be different depending on device/monitor. Maybe we should use the viewport height instead

Reply:
the viewport height is constant. could you make an example on how to use it in this context?

Reply:
`window.innerHeight` will return the height of the viewport. It is only constant to the currently visible viewport of the browser, which is different depending on the size

Reply:
for example:
`if (window.scrollY <= window.innerHeight * 0.5) {}`
to only show it if the user scrolled half way through the window.
But I just checked on their website, it looks to me like they also hardcoded it. I guess we can keep it like that :)